pycbc/_version.py

```diff
@@ -75,40 +76,82 @@ def get_lal_info(module, lib_glob):
     return version_str
 
 
-class Version(argparse.Action):
-    """Subclass of argparse.Action that prints version information for PyCBC,
-    LAL and LALSimulation.
+class PyCBCVersionAction(argparse._StoreAction):
+    """Subclass of argparse._StoreAction that prints version information for
+    PyCBC, and for LAL and LALSimulation depending on an integer variable.
+    Can be supplied without the option
     """
-    def __init__(self, nargs=0, **kw):
-        super(Version, self).__init__(nargs=nargs, **kw)
-
-    def __call__(self, parser, namespace, values, option_string=None):
-        import pycbc
+    default_help = (
+        'Display PyCBC version information and exit. '
+        'Can optionally supply a modifier integer to control the '
+        'verbosity of the version information. 0 and 1 are the '
+        'same as --version; 2 provides more detailed PyCBC library '
+        'information; 3 provides information about PyCBC, '
+        'LAL and LALSimulation packages (if installed)'
+    )
```

Comment:
What is the idea behind 0 and 1 both being equivalent to no option?

Reply:
Ah let me guess, `--version --version` is equivalent to `--version 2`, so 0 and 1 are both equivalent to a single `--version`?

Reply:
Thats what I _wanted_ to happen, (similar to the verbose action), but argparse's version and count asctions did not play nicely with one another when I tried to combine them.

Basically telling it to count, but then also to exit during the argument parsing stage wasn't possible

Reply:
Ok, so in some sense the specific choise of values is a slight inconvenient due to limitations in argparse?

Reply:
yeah, though I can put it back so that 0,1,2 give the different options if preferred

Reply:
I guess the only odd thing is zero being a valid option (since, to me, `--version 0` sounds like "give me version info, but none of it"). But we can leave it as is.